msyncd/AccountsHelper.cpp

```diff
@@ -42,7 +44,8 @@ AccountsHelper::AccountsHelper(ProfileManager &aProfileManager, QObject *aParent
     QObject::connect(iAccountManager, SIGNAL(accountUpdated(Accounts::AccountId)),
                      this, SLOT(slotAccountUpdated(Accounts::AccountId)));
 
-    registerAccountListeners();
+    // load accounts after return from contructor
```

Comment:
Nitpick: not actually true (not after ctor, but when returning to event loop), also more important would be the reason.